specification/resourcegraph/resource-manager/readme.md

```diff
@@ -50,13 +49,15 @@ These settings apply only when `--tag=package-2019-04` is specified on the comma
 input-file:
   - Microsoft.ResourceGraph/stable/2019-04-01/resourcegraph.json
 ```
+
 ### Tag: package-2018-09-preview
 
 These settings apply only when `--tag=package-2018-09-preview` is specified on the command line.
 
 ``` yaml $(tag) == 'package-2018-09-preview'
 input-file:
-- Microsoft.ResourceGraph/preview/2018-09-01-preview/resourcegraph.json
+  - Microsoft.ResourceGraph/preview/2018-09-01-preview/resourcegraph.json
```

Comment:
I'm not sure but should we refer to 2019-04-01 for this? Not sure how the SDK's work with different versions.

Reply:
We have both 2019-04-01 and 2018-09-01 since we still need to support the old version.